examples/run_grpo_random_dataset.py

```
@@ -0,0 +1,272 @@
+            )
+
+            input_generator = get_sequence_length_generator(
+                input_len_or_input_len_generator
+            )
+            data_config["input_len_or_input_len_generator"] = input_generator
+        else:
+            data_config["input_len_or_input_len_generator"] = (
+                input_len_or_input_len_generator
+            )
+    else:
+        assert False, "input_len_generator_cfg must be provided"
```

Ruff (0.14.3)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)